mcux/mcux-sdk/components/osa/fsl_os_abstraction_zephyr.c

```diff
@@ -192,8 +186,6 @@ void *OSA_MemoryAllocateAlign(uint32_t memLength, uint32_t alignbytes)
 
 void OSA_MemoryFreeAlign(void *p)
 {
-    OSA_ASSERT(p != NULL, "Trying to free memory pointing to NULL\r\n");
```

Comment:
why are you removing these

Reply:
already have the check of "if (p != NULL)", so the OSA_ASSERT is redundant.
fsl_os_abstraction_free_rtos.c also don't have this check.
And keep same as downstream, which is verified by QA since Zephyr V3.7